### PR TITLE
Merge VolPy 

### PR DESCRIPTION
**Pull request by caichangjia**
# Description

Merge VolPy into CaImAn

1.  Add multiple new files for analyzing voltage imaging data
2.  Allow to load multiple .tif files
3.  Add argument masktasksperchild in setup_cluster
4.  Add compute_BL function which compute baseline function with less memory consumption

### CODE REVIEW

**Comment by pgunn** [comment]
Hi,
Could you lowercase Volparams.py and spikePursuit.py?

File: caiman/source_extraction/volpy/mrcnn/config.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
+"""
+Mask R-CNN
+Base Configurations class.
+
+Copyright (c) 2017 Matterport, Inc.
+Licensed under the MIT License (see LICENSE for details)
```
Comment (pgunn) [inline caiman/source_extraction/volpy/mrcnn/config.py]: References a license file that's not included. This needs to be fixed.

**Comment by pgunn** [comment]
I wish mrcnn had a conda package of its own so we wouldn't end up vendoring it with this diff. Particularly problematic as years later, if mrcnn has some bugfixes and other improvements, we'll still likely have this same version still stuck in our codebase that we don't maintain, pay attention to, or understand.

Is there any other way to do this?

**Comment by epnev** [comment]
@caichangjia Add a reference to the preprint on the README file

File: caiman/source_extraction/volpy/mrcnn/config.py (hunk 1 above)
Comment (caichangjia) [reply caiman/source_extraction/volpy/mrcnn/config.py]: So Pat, what should we do here?

**Pull request merged**